### ISSUE TITLE
Use with expo/create-react-native-app

### ISSUE THREAD

**Issue by tslater** [matc4/react-native-svg-uri#66]
Expo already includes/supports react-native-svg. Is there anyway this package could be updated to support Expo?

**Comment by matc4** [comment]
Do you know what needs to support Expo?

**Comment by Noitidart** [comment]
I don't think this is possible. You can't use any libs that require `react-native link` last I heard. Things might have changed though.

**Comment by artufimtcev** [comment]
This PR seems to resolve this issue: https://github.com/vault-development/react-native-svg-uri/pull/111

Would be nice to have it included

**Comment by sandersjj** [comment]
I can confirm that this solution works for me with expo 32

**Comment by msand** [comment]
You can also just include that one file in your project as well.
